message_converter/src/converter_node.cpp

```diff
@@ -98,20 +37,23 @@ void SwiftNavRover::baselinePositionCallback(const libsbp_ros_msgs::MsgBaselineN
     int tow = msg->tow;
     int h_accuracy = msg->h_accuracy;
     int v_accuracy = msg->v_accuracy;
-    boost::array<double, 9> covariance = {{h_accuracy, 0, 0, 0, h_accuracy, 0, 0, 0, v_accuracy}};
+    boost::array<double, 9> covariance = {{h_accuracy/1000.00, 0, 0, 0, h_accuracy/1000.00, 0, 0, 0, v_accuracy/1000.00}};
     int n_sats = msg->n_sats;
     int fixed_mode = (msg->flags) & (0b00000111);
+    SwiftNavRover::positioning_mode_ = fixed_mode;
     ros::Time t = ros::Time::now();
     if(n_sats == 0){
```

Comment:
**[OPTIONAL]** May want to add a `ROS_WARN_THROTTLE` to flag there are no satellites in view.